extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/ApplyMemoryDecorator.java

```diff
@@ -0,0 +1,44 @@
+package io.quarkus.kubernetes.deployment;
+
+import io.dekorate.deps.kubernetes.api.model.ContainerFluent;
+import io.dekorate.deps.kubernetes.api.model.Quantity;
+import io.dekorate.kubernetes.decorator.ApplicationContainerDecorator;
+import io.dekorate.kubernetes.decorator.Decorator;
+import io.dekorate.kubernetes.decorator.ResourceProvidingDecorator;
+
+//TODO: To Remove when #653 (https://github.com/dekorateio/dekorate/pull/653) will be merge in dekorate and new version bumped in Quarkus
+public class ApplyMemoryDecorator extends ApplicationContainerDecorator<ContainerFluent<?>> {
```

Comment:
Why don't we use the upstream decorators?

Reply:
Hello @iocanel 

Because as for now, the generated resources for memory limits/requests is named "**mem**" and not "**memory**" in dekorate.
This was the goal of my PR: https://github.com/dekorateio/dekorate/pull/653 you merged into dekorate some days ago.

Until this fix is incorporated into a dekorate release and the new version bumped into Quarkus, we have to use that custom decorator.
Once done, of course, I will correct to use the upstream decorator.

Reply:
Hello @iocanel 

Now that upstream fix have been integrated into Quarkus, I remove the custom decorator for "requests memory".